src/CMakeLists.txt

```diff
@@ -147,19 +147,14 @@ function(ov_tokenizers_link_sentencepiece TARGET_NAME)
 endfunction()
 
 if(BUILD_FAST_TOKENIZERS)
-  set(EXTERNAL_PROJECT_SOURCE_DIR ${CMAKE_BINARY_DIR}/_deps/fast_tokenizer/src)
-  set(EXTERNAL_PROJECT_BINARY_DIR ${CMAKE_BINARY_DIR}/_deps/fast_tokenizer/build)
-  set(EXTERNAL_PROJECT_SUBBUILD_DIR ${CMAKE_BINARY_DIR}/_deps/fast_tokenizer/sub-build)
   set(WITH_PYTHON OFF CACHE BOOL "")
 
   FetchContent_Declare(
     fast_tokenizer
-    URL      https://github.com/PaddlePaddle/PaddleNLP/archive/refs/tags/v2.6.1.tar.gz
-    URL_HASH SHA256=10e3489bc91e938c449a0448fa719e4536803ed6b1c1c95b3402430d6a8a221a
-    PATCH_COMMAND git --git-dir=${EXTERNAL_PROJECT_SOURCE_DIR} apply --ignore-whitespace "${CMAKE_CURRENT_LIST_DIR}/icu4c.patch"
-    SOURCE_DIR ${EXTERNAL_PROJECT_SOURCE_DIR}
-    BINARY_DIR ${EXTERNAL_PROJECT_BINARY_DIR}
-    SUBBUILD_DIR ${EXTERNAL_PROJECT_SUBBUILD_DIR}
+    GIT_REPOSITORY https://github.com/PaddlePaddle/PaddleNLP
+    GIT_TAG v2.6.1
+    GIT_SHALLOW ON
+    PATCH_COMMAND git apply --ignore-whitespace "${CMAKE_CURRENT_LIST_DIR}/icu4c.patch"
```

Comment:
why do we need this change?
This requires git, while previous solution does not.
Secondly, git clone is longer than archive downloading